src/lib/auth.ts

```diff
@@ -165,6 +164,14 @@ export const authenticate = (devMode: boolean) => {
 
             resolve(user);
           });
+      })
+      .catch((e) => {
```

Comment:
This was once thing the unit tests reveal - if `getPortalJWTWithBearerToken` or `getClassInfo` are rejected, the `authentication` promise never resolves! So it seems like I need to pass rejections along like this